CRM/Contact/BAO/Query.php

```diff
@@ -4908,6 +4929,24 @@ public function searchQuery(
     return $dao;
   }
 
+  /**
+   * Create and query the db for a contact search.
+   *
+   * @return CRM_Core_DAO
+   */
+  public function alphabetQuery() {
```

Comment:
I'll put up an extraction on this as a separate PR cos I'd decided to do that too